internal/promotion/runner/builtin/git_cloner_test.go

```diff
@@ -280,6 +280,59 @@ func Test_gitCloner_validate(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "duplicate as aliases",
```

Comment:
I would also suggest moving this test case higher up so that we finish with the kitchen sink.

```suggestion
			name: "duplicate aliases",
```